src/modules/_partials/layouts/DashboardLayout/index.tsx

```diff
@@ -55,15 +55,12 @@ const DashboardLayout = ({
   const { mode } = useTheme();
   return (
     <Flowbite theme={{ theme }}>
-      <div className="relative flex w-full overflow-x-hidden bg-brand-blue-light dark:text-gray-50 dark:bg-gray-900 h-screen overflow-y-auto">
+      <div className="relative flex w-screen overflow-x-scroll bg-brand-blue-light dark:text-gray-50 dark:bg-gray-900 h-screen overflow-y-auto">
```

Comment:
@bizip To achieve full width, you should use the w-full class instead of w-screen. The w-screen class is designed to match the width of the screen, including any scrollbars or browser UI elements, which might not be the desired behavior.

Let's avoid horizontal scroll wherever possible.